src/lib/core/ripple/ripple.spec.ts

```diff
@@ -180,6 +180,47 @@ describe('MdRipple', () => {
       expect(pxStringToFloat(ripple.style.width)).toBeCloseTo(2 * expectedRadius, 1);
       expect(pxStringToFloat(ripple.style.height)).toBeCloseTo(2 * expectedRadius, 1);
     });
+
+    it('create ripple with correct position when page is scrolled', () => {
+      let elementTop = 600;
+      let elementLeft = 750;
+      let pageScrollTop = 500;
+      let pageScrollLeft = 500;
+      let left = 50;
+      let top = 75;
+
+      // Add a very large element to make the page scroll
+      let veryLargeElement: HTMLElement = document.createElement('div');
```

Comment:
You don't need the type here; `document.createElement('div');` will return an `HTMLDivElement`

(typescript understands string literals in its type system so it can know things like `document.createElement` or `addEventListener` can have different arguments based on the string based to them)